tests/ui/specs/VirtualList/VirtualGridList/With5-way/VirtualGridList-GT-28584-specs.js

```diff
@@ -0,0 +1,53 @@
+const {expectFocusedItem} = require('../../VirtualList-utils');
+const Page = require('../VirtualGridListPage');
+
+describe('Navigate with 5-way', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	it('should navigate from and to last item via 5-way [GT-28584]', function () {
+		// Step 3-1:  Knobs > VirtualGridList > dataSize > 17
```

Comment:
When I set the dataSize to 17, Item locations are different from the original TC.

![image](https://user-images.githubusercontent.com/34676178/116043574-cda92e80-a6aa-11eb-9e39-c02be96c4078.png)
How about using Item13 and Item04 after setting dataSize to 14?

Reply:
Suggested change.